### DISCUSSION TITLE
Not enough time to field account information

### DISCUSSION THREAD

**Discussion by NikoWeb99** [cryinkfly/Autodesk-Fusion-360-for-Linux#335] [General]
I've just installed fusion360 with the script! this project is amazing and i'm excpecting a lot because I like fusion and Linux.

everything seems ok but when I launch fusion the lauch windows start and I can see the login windows but I don't have time tu field it, it disappear quickly and nothing happens.

could help me?

tks

**Comment by LuZo101** [comment]
there is actually a problem that fusion want pass to a login browser window... this bug still not fixed now. but i thin they are working on it ...

**Comment by NikoWeb99** [comment]
Thanks for your answer. I thougt this issue with fusion loin browser has been fixed.
For my case I don't know Why but after several attemps  I was able to login.
But no it's very slow and not stable. Sometimes I have black windows and all systèm crashes.
I précise it's a new systèm ( mint), I réinstall it Just yesterday to try fusion in a fresh install.

Tks
